json_object.c

```diff
@@ -644,6 +644,18 @@ struct fjson_object* fjson_object_new_int64(int64_t i)
 	return jso;
 }
 
+uint32_t fjson_object_get_uint(struct fjson_object *jso)
+{
+	int64_t cint = fjson_object_get_int64(jso);
+    if (errno) return 0;
```

Comment:
pls change the indention in the function to tab (HT) - that will also make the travis errors go away.

Reply:
Switched to tabs.